src/core/server.cc

```diff
@@ -218,6 +219,75 @@ class InferContext final
   }
 };
 
+// For now, this context ExecuteRPC in the same way as InferContext, except that
```

Comment:
If this code is (mostly) the same then should use a common function. How is the bi-directional streaming being used here?

Reply:
It is the same as the ExecuteRPC(), I didn't know if I will need to change the way to acquire servable if we want the requests to be processed by the same servable instance. If no changes are needed, I agree that we can extract it as a common function.

The bidirectional streaming part is in `BidirectionalStreamingLifeCycle.h`